envoy/gateway-endpoints.example.yaml

```diff
@@ -0,0 +1,47 @@
+# TODO_IMPROVE(@commoddity): Update schema URL to point to main branch once `yaml-data-source` branch is merged into main.
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildwithgrove/path-auth-data-server/refs/heads/yaml-data-source/yaml/gateway-endpoints.schema.yaml
+
+# The above schema URL may be used to validate this file using the `yaml-language-server` VSCode extension.
+# See: https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml
+
+# This file shows examples of how to configure PATH gateway endpoints using a YAML file as the PADS data source.
+# These files will be converted to the proto.GatewayEndpoint struct expected by the PATH's Go External Auth Server.
+#
+# For details on the expected proto format, see the definition in  `envoy/auth_server/proto/gateway_endpoint.proto`.``
```

Comment:
Is there a copy of this file in another PR? Any way to consolidate the two so there's just one version of the "source of truth"?

Reply:
It also exists in PADS; I could just link to it.

Reply:
Let's do it.